docs/data/stashcache/run-stashcache-container.md

```diff
@@ -19,14 +19,99 @@ This document outlines how to run a cache in a Docker container.
 Before Starting
 ---------------
 
-Before starting the installation process, consider the following points:
+Before starting the installation process, consider the following requirements:
```

Comment:
I think we use "points" everywhere in the docs repo. I think we may as well just keep that wording?

Reply:
We're not consistent. For example, we use "following requirements" in compute-element/hosted-ce.md and "following points" in compute-element/install-htcondor-ce.md.

We use "requirements" in the RPM-based cache install doc so I wanted to be consistent with that.

Reply:
Mmk, fine by me. May as well change the origin docs while we're at it so they match